shared/verify-sonarqube-passed.sh

```diff
@@ -2,15 +2,30 @@
 FULL_URL="https://sonarcloud.io"
```

Comment:
There has to be one for each project in the build. But this might be a good target for simplification at some point though it would mean the project could not be broken up into separate repos as easily later.

Reply:
Or perhaps a single script that they could each reference? In either case, it's not a blocker right now.